scripts/copy-kubeconfig.yaml

```diff
@@ -0,0 +1,61 @@
+---
+- hosts: kube-master[0]
+  gather_facts: no
+  become: yes
+  tasks:
+  - fetch:
+      src: "/etc/kubernetes/ssl/{{ item }}.pem"
+      dest: "{{ playbook_dir }}/kubectl/{{ item }}.pem"
+      flat: True
+    with_items:
+      - admin-{{ inventory_hostname }}-key
+      - admin-{{ inventory_hostname }}
+      - ca
+  - name: export hostname
+    set_fact:
+      kubectl_name: "{{ inventory_hostname }}"
+
+- hosts: localhost
+  connection: local
+  vars:
+    kubectl_name: "{{ hostvars[groups['kube-master'][0]].kubectl_name }}"
+    cluster_name: "{{ hostvars[groups['kube-master'][0]].cluster_name }}"
+    kube_apiserver_port: "{{ hostvars[groups['kube-master'][0]].kube_apiserver_port }}"
+    system_namespace: "{{ hostvars[groups['kube-master'][0]].system_namespace }}"
+  tasks:
+  - name: "check if context admin@{{ cluster_name }} exists"
+    command: kubectl config get-contexts admin@{{ cluster_name }}
```

Comment:
You got me, I already had installed kubectl locally. We can not just copy it from hyperkube or master becuase the host might not be the same OS and CPU architecture as the local computer.

The safest is probably to download it like this:

```yaml
- name: download kubectl
  get_url:
    url: https://storage.googleapis.com/kubernetes-release/release/{{ kubernetes_version }}/bin/{{ ansible_system|lower }}/{{ ansible_architecture|lower }}/kubectl
    dest: /usr/local/bin/kubectl
    mode: 0755
```